pkg/cluster/cluster.go

```diff
@@ -165,11 +165,11 @@ func (c *Cluster) createMachineRunArgs(machine *Machine, name string, i int) []s
 	runArgs := []string{
 		"-it", "-d", "--rm",
 		"--name", name,
-		"--hostname", machine.Hostname(),
```

Comment:
@dlespiau Now this is more interesting. One of these f*cks up SSH :D No idea yet which one, probably the hostname. without these ssh was unable to start exiting with 255. I run out of time to investigate this further.